ajax/add-ftp-user.php

```diff
@@ -0,0 +1,37 @@
+<?php
```

Comment:
Nick is absolutely right, this AJAX endpoint is not secure at all, it WILL get crawled and found and abused. To be honest, there is no need at all for AJAX here, this POST should be handled within the main page. (Take a look at the auth/join file for an example)

Reply:
moved it to the file